src/views/NoteView.vue

```diff
@@ -1,5 +1,165 @@
 <template>
-  <h1>Note View</h1>
+  <div class="note-view">
+    <div class="header-controls">
+      <div class="note-info">
+        <div class="note-channel">
+          <span>channel: </span>
+          <span class="note-channel">{{ note.channel }}</span>
+        </div>
+        <div class="note-tag">
+          <span>tag: </span>
+          <span class="note-tag">{{ note.tag }}</span>
+        </div>
+      </div>
+      <div class="note-actions">
+        <button class="note-action-button">ホームへ戻る</button>
+        <button class="note-action-button">編集</button>
+      </div>
+    </div>
+    <h1 class="note-title">{{ note.title }}</h1>
+    <div class="note-body" v-html="renderedMarkdown"></div>
+  </div>
 </template>
 
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import type { NoteSummary } from '@/types/api';
+import { ref, computed } from 'vue';
+import { marked } from 'marked';
+import DOMPurify from 'dompurify';
+
+const renderedMarkdown = computed(() => {
+  if (!note.value.summary) {
+    return '';
+  }
+  // markedでMarkdownをHTMLに変換し、DOMPurifyでサニタイズする
+  const rawHtml = marked(note.value.summary) as string;
+  return DOMPurify.sanitize(rawHtml);
+});
+
+const note = ref<NoteSummary>({
+  id: '0197882d-208b-7c5a-bf60-89eafb904109',
+  permission: 'limited',
+  channel: 'channel-1',
+  tag: 'markdown, development',
+  title: 'Markdownの使い方',
+  summary:
+    'Markdown は色々な事が出来ます。例えば、**太字**や*斜体*、[リンク](https://example.com)など。\nまた、コードブロックも使えます。\n```javascript\nconsole.log("Hello, World!");\n```\nさらに、リストも作成できます。\n- アイテム1\n- アイテム2\n- アイテム3',
+  createdAt: 1750494000,
+  updatedAt: 1750494000,
+});
```

Comment:
ストアから内容を引っ張るようにしましょう。

```suggestion
import { useNotesStore } from '@/stores/notes';
const notesStore = useNotesStore();

const note = computed(() => notesStore.currentNote);
```